docs/content/latest/api/ysql/exprs/window_functions/_index.md

```diff
@@ -2,7 +2,7 @@
 title: YSQL window functions
 linkTitle: Window functions
 headerTitle: Window functions
-description: Window functions operate on a row set that's defined, in general, as an ordered window within a containing restriction defined by a subquery. They calculate a value for each row in the window by consulting values from other rows in the window.
+description: This major section escribes the synatx and semantics of all of the Window functions that YSQL supports. These operate on a row set that's defined, in general, as an ordered window within a containing restriction defined by a subquery. They calculate a value for each row in the window by consulting values from other rows in the window.
```

Comment:
```suggestion
description: This section covers the syntax and semantics for all supported window functions in YSQL. 
```

- length of 336 characters is way too long — Google will ignore or rewrite.  Try to keep this under 160 characters.

Reply:
A meta description length of 336 characters is too long — Google will ignore, rewrite it, or use part of the first sentence on the page.  Try to keep this under 160 characters.

Reply:
Done.